torchvision/ops/boxes.py

```diff
@@ -325,13 +325,13 @@ def complete_box_iou(boxes1: Tensor, boxes2: Tensor, eps: float = 1e-7) -> Tenso
 
     diou, iou = _box_diou_iou(boxes1, boxes2, eps)
 
-    w_pred = boxes1[:, 2] - boxes1[:, 0]
-    h_pred = boxes1[:, 3] - boxes1[:, 1]
+    w_pred = boxes1[:, None, 2] - boxes1[:, None, 0]
+    h_pred = boxes1[:, None, 3] - boxes1[:, None, 1]
 
     w_gt = boxes2[:, 2] - boxes2[:, 0]
     h_gt = boxes2[:, 3] - boxes2[:, 1]
 
-    v = (4 / (torch.pi**2)) * torch.pow((torch.atan(w_gt / h_gt) - torch.atan(w_pred / h_pred)), 2)
+    v = (4 / (torch.pi**2)) * torch.pow(torch.atan(w_pred / h_pred) - torch.atan(w_gt / h_gt), 2)
```

Comment:
Was this a bug before?

Reply:
Nevermind, it's a squared function afterwards, doesn't matter :-)

Reply:
No that's just a styling/nit thingy. The actual bug was on [this](https://github.com/pytorch/vision/pull/6338/files#diff-c49f3b58b42045e50c2157a195bfb83f34cc0d1842dc92aa0e9aa402499dcd88R378-R380) line. Thanks for providing the reference implementation, it really helped me debug it. 😃